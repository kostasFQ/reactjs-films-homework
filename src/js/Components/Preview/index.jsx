import React from 'react';
import PropTypes from 'prop-types';
import styles from './preview.scss';

const Preview = (props) => {
  const { overview } = props;
  return (
    <div className={styles.container}>
      <div className={styles.buttonsBlock}>
        <div>{overview}</div>
      </div>
    </div>
  );
};

export default Preview;

Preview.propTypes = {
  overview: PropTypes.string.isRequired,
};
