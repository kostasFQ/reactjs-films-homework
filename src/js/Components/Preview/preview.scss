.container {
  position: relative;
  width: 100%;
}

.previewField {
  text-align: left;
  color: white;

  @media only screen and (max-width: 600px) {
    margin-left: 5%;
    width: 100%;
    display: flex;
    position: absolute;
    background-color: rgba(0, 0, 0, 0.6);
  }
}
