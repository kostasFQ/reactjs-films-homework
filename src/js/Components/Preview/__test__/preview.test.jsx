import React from 'react';
import renderer from 'react-test-renderer';
import Preview from '../index';

test('test component render', () => {
  const render = renderer.create(
    <Preview preview="string" />,
  );
  const component = render.toJSON();

  expect(component).toMatchSnapshot();
  expect(component.type).toBe('div');
});
