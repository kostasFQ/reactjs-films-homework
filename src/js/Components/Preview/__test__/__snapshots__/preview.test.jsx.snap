// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test component render 1`] = `
<div
  className="container"
>
  <div
    className="buttonsBlock"
  >
    <div />
  </div>
</div>
`;
